### PR TITLE
[Doc Improvement][Payload added in Adaptive Card TaskInfo.card section][2758454]

### PR DESCRIPTION
**Pull request by v-sdhakshina**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/158cf37269d945756967fa0eee41280e9742cc5e/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-tabs.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b83e7136-68f3-42dd-8af8-afe4cde0c37d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md [hunk 1]
```diff
@@ -82,6 +82,60 @@ Pass your result as the first parameter. Teams invokes `submitHandler` where `er
...
+{
+  "task": {
+    "type": "continue",
+    "value": {
+      "title": "Title",
+      "height": "height",
+      "width": "width",
+      "url":null,
+
+  "card": "Adaptive Card or Adaptive Card bot card attachment"
```
Comment (Wajeed-msft) [inline msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md:97]: Formatting seems to be incorrect, also comma is missing after "card" section.

File: msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md [hunk 2]
```diff
@@ -82,6 +82,60 @@ Pass your result as the first parameter. Teams invokes `submitHandler` where `er
...
+            };
+
+taskInfo.title = "Task Module Demo";
+taskInfo.height = "medium";
+taskInfo.width = "medium";
+
+taskInfo.card =  "Adaptive card or adaptive card attachment code";
+taskInfo.fallbackUrl = taskInfo.url;
+
+if (type.includes('bot'))
```
Comment (Wajeed-msft) [inline msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md:127]: Please fix the formatting.

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a58553b39b03e1f97ded663d9b3632dc5e87bc6/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-tabs.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/414eb972-2641-46a8-b8d5-02c56b87ff23) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/dc34551e6a701553ef3b0b3c04d6334c2efce82d/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-tabs.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0ff6e69e-366a-4af2-818e-dccabecdfe27) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1430c228f743caa39907f25789dc6df4a10ec25f/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-tabs.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aa0f5cca-df66-4fad-9a29-651a0a710ec6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Comment by v-sdhakshina** [comment]
@tomlm Request you to please review and approve.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8f4ef9375dad91af3fcc46c12e7e91b660f9bbc9/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-tabs.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/63fabae7-5d5a-4981-a327-2170a5581503) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d869fa491d7e57b3e20443a8dce99ba6c4361ee4/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Ftask-modules-tabs.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9a68297c-9c90-4e26-bd03-4e19df8fabd7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**